packages/config/index.ts

```diff
@@ -0,0 +1,27 @@
+import * as fs from 'fs'
+import * as path from 'path'
+import dotenv from 'dotenv'
+
+dotenv.config({ path: '../../.env' })
+
+// config.json file at project root
+export const DEFAULT_CONFIG_PATH = '../../config.json'
```

Comment:
Before merging this PR, we must ensure all monitors scripts in the bridge are updated to handle this change + we must merge both the PRs together to ensure it doesn't break when cron runs.

Reply:
Oh I meant having a PR in the Bridge UI ready - to handle the change in the architecture in this PR.